### PR TITLE
GitHub: Fixed XAML formatting checker

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Resolved / Related Issues

_None_

### Steps used to test these changes

_None_

### CODE REVIEW

**Comment by 0x5bfa** [comment]
@yaira2 Fixed the xaml formatting checker and the xaml formatter. Up for review & merge.

I'll run the xaml formatter later in another PR after this is merged since issue_comment event is fired based on the default branch (the main branch) while pull_request event is based on the head branch.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,10 +49,7 @@ jobs:
 
     if: github.repository_owner == 'files-community'
 
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: pwsh
+    runs-on: windows-latest
```
Comment (Lamparter) [inline .github/workflows/ci.yml:52]: What's the change for?

Comment (Lamparter) [reply .github/workflows/ci.yml:52]: Just out of interest 😅

Comment (0x5bfa) [reply .github/workflows/ci.yml:52]: Just reckoned why it should be Ubuntu while the default shell is specified to be pwsh.

Comment (Lamparter) [reply .github/workflows/ci.yml:52]: Isn't ubuntu far cheaper?

Comment (0x5bfa) [reply .github/workflows/ci.yml:52]: In what aspect? Set up finishes in a matter of seconds.

Comment (Lamparter) [reply .github/workflows/ci.yml:52]: It costs less credits to use Linux, and considering that Files uses Windows for most of its actions I'm sure it's not got loads left.

Comment (0x5bfa) [reply .github/workflows/ci.yml:52]: Anyways, doesn't really matter.

Comment (yaira2) [reply .github/workflows/ci.yml:52]: Whichever is quicker and uses the least resources is preferred.

File: src/Files.App/Views/MainPage.xaml [hunk 2]
```diff
@@ -30,12 +30,8 @@
 				<ResourceDictionary Source="ms-appx:///Styles/DefaultGridSplitterStyle.xaml" />
 			</ResourceDictionary.MergedDictionaries>
 
-			<x:Boolean x:Key="True">True</x:Boolean>
-			<x:Boolean x:Key="False">False</x:Boolean>
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml:34]: Can this be done in a separate PR?

Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml]: Done.

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml]: Thank you

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yaira2) [reply .github/workflows/ci.yml:52]: As per our conversation on Discord, let's switch back to Linux.

Comment (Lamparter) [reply .github/workflows/ci.yml:52]: @heftymouse made a good point, I'd forgotten the credit limit only applies to private repos. Is windows that much quicker?

Comment (0x5bfa) [reply .github/workflows/ci.yml:52]: I haven't compared but the setup finishes 1 sec.

**Comment by yaira2** [comment]
LGTM
